src/components/visualizations/chart/highcharts/helpers.ts

```diff
@@ -72,14 +72,19 @@ export const isStacked = (chart: any) => {
 };
 
 export function getChartProperties(config: IChartConfig, type: VisType) {
-    const isBarType = isBarChart(type);
+    const isInvertedChart = isInvertedChartType(type);
```

Comment:
the configurations for the x-axis and y-axis must be switched for both bar chart and bullet chart.